### PR TITLE
Style css2

### PR DESCRIPTION
**Pull request by sebastian-nehrdich**
This is to merge the changes that we made together with Lars over the last weeks on the frontend.

### CODE REVIEW

File: src/router.js [hunk 1]
```diff
@@ -193,12 +258,13 @@ class BNRouter {
...
   static selectTab(tabName) {
-    const vaadinTabs = getMainLayout().querySelectorAll('vaadin-tab');
-    vaadinTabs.forEach(item => {
-      item.removeAttribute('selected');
-    });
+  //this is commented out because we need to change the menu bar first
+  // const vaadinTabs = getMainLayout().querySelectorAll('vaadin-tab');
+  // vaadinTabs.forEach(item => {
+  //   item.removeAttribute('selected');
+  // });
```
Comment (ayya-vimala) [inline src/router.js:265]: I think this whole function can probably go.

File: src/router.js [hunk 2]
```diff
@@ -193,12 +258,13 @@ class BNRouter {
...
 
-    vaadinTabs[TABS_IN_ORDER.indexOf(tabName)].setAttribute('selected', 'true');
+  // vaadinTabs[TABS_IN_ORDER.indexOf(tabName)].setAttribute('selected', 'true');
```
Comment (ayya-vimala) [inline src/router.js:267]: idem

File: src/views/static/community/community-view.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
+import { customElement, html, LitElement } from 'lit-element';
```
Comment (ayya-vimala) [inline src/views/static/community/community-view.js:1]: Is this page needed at all? I think it is not being used?

File: src/views/static/about/about-view.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import { customElement, html, LitElement } from 'lit-element';
```
Comment (ayya-vimala) [inline src/views/static/about/about-view.js:1]: Is this page being used at all?

File: src/menu-data.js [hunk 5]
```diff
@@ -0,0 +1,95 @@
+export const navMenuDataMain = [
+  {
+    text: 'About',
+    url: 'about',
```
Comment (ayya-vimala) [inline src/menu-data.js:4]: I don't think yuo can actually select this and there is nothing on that page

File: src/menu-data.js [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+      },
+      {
+        text: ' History',
+        url: 'history',
+      },
+    ],
+  },
+  {
+    text: 'Community',
+    url: 'community',
```
Comment (ayya-vimala) [inline src/menu-data.js:18]: Idem

File: src/menu-data.js [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+    text: 'News',
+    url: 'news',
+  },
+  {
+    text: 'Guidelines',
+    url: 'guidelines',
+  },
+  {
+    text: 'Activities',
+    url: 'Activities',
```
Comment (ayya-vimala) [inline src/menu-data.js:40]: idem

File: src/menu-data.js [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+      },
+    ],
+  },
+  {
+    text: 'Publications',
+    url: 'publications',
+  },
+  {
+    text: 'Database',
+    url: 'visual',
```
Comment (ayya-vimala) [inline src/menu-data.js:62]: idem and this is never the right url for database anyway.

File: src/router.js [hunk 9]
```diff
@@ -1,13 +1,21 @@
 import { Router } from '@vaadin/router';
 
 import './views/static/home/home-view.js';
+import './views/static/about/about-view.js';
```
Comment (ayya-vimala) [inline src/router.js:4]: I think this is obsolete

File: src/router.js [hunk 10]
```diff
@@ -1,13 +1,21 @@
...
+import './views/static/introduction/introduction-view.js';
 import './views/static/history/history-view.js';
-import './views/static/people/people-view.js';
+import './views/static/community/community-view.js';
```
Comment (ayya-vimala) [inline src/router.js:7]: idem

File: src/router.js [hunk 11]
```diff
@@ -1,13 +1,21 @@
...
 import './views/static/institutions/institutions-view.js';
-import './views/static/activities/activities-view.js';
-import './views/static/publications/publications-view.js';
+import './views/static/people/people-view.js';
+import './views/static/news/news-view.js';
 import './views/static/guidelines/guidelines-view.js';
+import './views/static/activities/activities-view.js';
```
Comment (ayya-vimala) [inline src/router.js:12]: idem

File: src/router.js [hunk 12]
```diff
@@ -1,13 +1,21 @@
...
 import './views/static/events/events-view.js';
+import './views/static/projects/projects-view.js';
+import './views/static/presentations/presentations-view.js';
+import './views/static/publications/publications-view.js';
+import './views/static/tools/tools-view.js';
```
Comment (ayya-vimala) [inline src/router.js:17]: I'm pretty sure this does not exist.

File: src/router.js [hunk 13]
```diff
@@ -42,71 +50,127 @@ const ROUTES = [
     component: 'home-view',
     action: () => {
       BNRouter.selectTab(TABS.HOME);
-      switchNavbarLayout(true);
+      switchNavbarLayout(true, true);
+    },
+  },
+  {
+    path: '/about',
```
Comment (ayya-vimala) [inline src/router.js:57]: This can go

File: src/router.js [hunk 14]
```diff
@@ -42,71 +50,127 @@ const ROUTES = [
...
     animate: true,
     component: 'history-view',
     action: () => {
-      switchNavbarLayout(true);
+      switchNavbarLayout(false, true);
     },
   },
   {
-    path: '/people',
+    path: '/community',
```
Comment (ayya-vimala) [inline src/router.js:81]: this can go

File: src/router.js [hunk 15]
```diff
@@ -42,71 +50,127 @@ const ROUTES = [
...
     path: '/guidelines',
     animate: true,
     component: 'guidelines-view',
     action: () => {
-      switchNavbarLayout(true);
+      switchNavbarLayout(false, true);
+    },
+  },
+  {
+    path: '/activities',
```
Comment (ayya-vimala) [inline src/router.js:121]: this can go

File: src/router.js [hunk 16]
```diff
@@ -136,7 +200,8 @@ const ROUTES = [
     action: () => {
       import('./views/data/data-view.js');
       BNRouter.selectTab(TABS.TIBETAN);
-      switchNavbarLayout(false);
+      switchNavbarLayout(false, false);
+      //      setFooterVisible(false);
```
Comment (ayya-vimala) [inline src/router.js:204]: remove comment

File: src/views/static/tools/tools-view.js [hunk 17]
```diff
@@ -0,0 +1,22 @@
+import { customElement, html, LitElement } from 'lit-element';
```
Comment (ayya-vimala) [inline src/views/static/tools/tools-view.js:1]: This page can probably go too

File: src/views/static/community/community-view.js (hunk 3 above)
Comment (sebastian-nehrdich) [reply src/views/static/community/community-view.js:1]: I think we want to use these pages in the future, yes. We need to find ways to link the menus to such pages when they are a 'head entry' in the menu. Doesn't work at the moment unfortunately.

File: src/views/static/about/about-view.js (hunk 4 above)
Comment (sebastian-nehrdich) [reply src/views/static/about/about-view.js:1]: Same as above, it _should_ be use but it isn't atm.

File: src/menu-data.js (hunk 5 above)
Comment (sebastian-nehrdich) [reply src/menu-data.js:4]: this is precisely the point. One remaining problem is that these items should become selectable. Orna has some content for these pages prepared. I tried a bit to make it work with vaadin-menus but so far no luck...

File: src/views/static/tools/tools-view.js (hunk 17 above)
Comment (sebastian-nehrdich) [reply src/views/static/tools/tools-view.js:1]: I opt to keep it. Its a template for wider functioanlity to be constructed in the (near) future.

Comment (sebastian-nehrdich) [reply src/views/static/tools/tools-view.js:1]: oh wait, this is under static. no that is wrong. we can indeed delete it. I will take care of this.

**Pull request merged**